manifests/d/dotPDN/PaintDotNET/4.3.12/dotPDN.PaintDotNET.installer.yaml

```diff
@@ -0,0 +1,100 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.7-3-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: dotPDN.PaintDotNET
+PackageVersion: 4.3.12
+InstallerType: zip
+Scope: machine
+UpgradeBehavior: install
+Protocols:
+- paintdotnet
+FileExtensions:
+- avif
+- bmp
+- dds
+- dib
+- gif
+- heic
+- jfif
+- jpe
+- jpeg
+- jpg
+- jxr
+- png
+- rle
+- tga
+- tif
+- tiff
+- wdp
+- webp
+- wmp
+ReleaseDate: 2022-08-20
+AppsAndFeaturesEntries:
+- UpgradeCode: '{04A40F40-A207-4B48-AED7-6AA532E43275}'
+  InstallerType: wix
```

Comment:
Won't this cause issues with the portable entry?

Reply:
I don't know. You can add this to per-installer entries.